Geometry_Engine/Modify/Rotate.cs

```diff
@@ -210,11 +226,29 @@ public static ICurve IRotate(this ICurve geometry, Point origin, Vector axis, do
         /***************************************************/
 
         public static ISurface IRotate(this ISurface geometry, Point origin, Vector axis, double rad)
+        {
+            return Rotate(geometry as dynamic, origin, axis, rad);
+        }
+
+        /***************************************************/
+        /**** Private Methods - Interfaces              ****/
+        /***************************************************/
+
+        [Description("Some objects have no use for origin, this method will make them calleble from the interface method.")]
```

Comment:
Should it be description or a comment?

Reply:
I figured that any comment which apply to the whole method might as well be a description.
But not sure there's any reason for it, could change it?

Reply:
Hmm, now I noticed the method is `private`. In such case it probs does not matter - I would only be concerned that the end user could see it at some point. But that is fine for now I guess 👍